src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/TypeInfoResolverFunctionalTests.cs

```diff
@@ -91,6 +92,24 @@ public static void AppendCharacterWhenSerializingField()
             Assert.Equal(originalObj.TestProperty, deserialized.TestProperty);
         }
 
+        [Fact]
+        public static void CorrectlyIgnorePropertyWithNewModifier()
```

Comment:
I thought I had left this feedback earlier, but I don't see it now. I don't believe this is the appropriate test suite for this type of test. Could you please move it to PropertyVisibilityTests.cs? Thanks

Reply:
Moved by amend: https://github.com/dotnet/runtime/commit/a9ca71cbaf5aa82304870d9622a57ebccbe5218f